server/infra/database/GrowerAccountRepository.ts

```diff
@@ -90,26 +104,15 @@ export default class GrowerAccountRepository extends BaseRepository<GrowerAccoun
       delete filterObject.device_identifier;
     }
 
-    // how to get all the parent and children ids for the org -- if they exist?
-    // Stakeholder api? or pass the org ids from the frontend?
     if (filterObject.organization_id) {
-      result.where(`${this.tableName}.organization_id`, 'in', [
-        filterObject.organization_id,
-      ]);
+      result.where(
+        `${this.tableName}.organization_id`,
+        'in',
+        filterObject.organization_id.split(','),
```

Comment:
I think we should make this an array right from Joi so that it is explicitly clear